### ISSUE TITLE
Failed to execute default Web Browser. Input/output error.

### ISSUE THREAD

**Issue by ftpronk** [Microsoft/vscode#45088]
- VSCode Version: 1.20.1 Commit: f88bbf9137d2
- OS Version: Linux Mint 18.3, Xfce Edition

Steps to Reproduce:

1. Install VSCode v1.20.1
2. Start VSCode via terminal or Application Finder. 
3. Install the "Language support for Java for Visual Studio Code" Extension
4. In an empty directory, create a new .java file and open it in vscode
5. A warning will pop-up: "Classpath is incomplete. Only syntax errors will be reported" -> Click on "More Information"
6. A system error dialog box will show: "Failed to execute default Web Browser. Input/Output error".

Browser-launching solution:

1. Start VSCode from the terminal with the **--verbose** flag
2. Open single .java file again. The same warning message should pop-up.
3. Click on "More Information". The browser will launch.

**Comment by ftpronk** [comment]
I beg to differ. The extension only happened to be the mechanism through which I encountered the bug, and therefore a way to describe it in a reproducible way.

Since then, I've noticed that when I click on a link, for example in any extension description, I get the same behaviour. So simply put, on my system, VS Code is unable to start a browser in normal mode.  But strangely enough, it can start a browser when in --verbose mode.

**Comment by Tyriar** [comment]
@ftpronk does the same thing happen when you open Help > Tips and Tricks?

**Comment by ftpronk** [comment]
Absolutely.  And, once more, the browser launches without issues when VSCode is started in the terminal with `code --verbose`.

**Comment by Tyriar** [comment]
@bpasero any ideas why opening the browser would only work when using `--verbose`?

**Comment by bpasero** [comment]
@Tyriar never saw that. Maybe https://ubuntuforums.org/showthread.php?t=2053643 ?

**Comment by ftpronk** [comment]
Might be a lead.  The browser launching from within VSCode does seem to work when I set Firefox as preferred browser.  But [the script on page 2 of the link](https://ubuntuforums.org/showthread.php?t=2053643&page=2&p=12275947#post12275947) doesn't help. And I generally don't have issues launching chromium using exo-open.

**Comment by neovatar** [comment]
I had the same effects on Ubuntu 16.04 with VSCode and Chromium. I debugged the code of the wrapper script (/usr/bin/chromium-browser) and the part that broke the wheels was sourcing customization files in /etc/chromium-browser/customizations/.  I my case, it was the customization code of the Adobe Flash plugin resulted in VSCode failing to launch chromium-browser. Once I removed the Adobe Flash plugin packages (adobe-flash-properties-gtk* adobe-flashplugin* flashplugin-installer*), the problem disappeared.
